### PR TITLE
Adding code fences to the example code.

### PR DESCRIPTION
**Pull request by ZehCnaS34**
A side note:

When I was trying the first example for the SQLite database, it raised an error when running

``` julia

db = connect(SQLite, "database.sqlite")

```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -160,76 +163,76 @@ any information required to make the connection. Different databases expect
...
-    using SQLite
-
-    connect(SQLite3, "db.sqlite3") do conn
-        println("Connected.")
-        error("Something went wrong.")
-    end
-
-    println("Disconnected.")
+connect(SQLite3, "db.sqlite3") do conn
+println("Connected.")
```
Comment (iamed2) [inline README.md]: Looks like you inadvertently lost some indentation here

**Comment by ScottPJones** [comment]
Might want to add an "n" to the word "feces" in the title! 😜

**Comment by ZehCnaS34** [comment]
Whoops lol. Thanks!

**Comment by ivarne** [comment]
Bump (this looks good to me)!

(Reviewers might want to look at https://github.com/JuliaDB/DBI.jl/pull/11/files?w=1 to see the non-whitesace changes.)

**Comment by iamed2** [comment]
Lost indentation in do block on line 184 still needs to be corrected

**Pull request closed without merging**